tempest/api/workloadmgr/base.py

```diff
@@ -3967,3 +3971,16 @@ def delete_secret_order(self, order_uuid):
         resp = self.order_client.delete_order(order_uuid)
         LOG.debug(f"resp {resp}")
         return resp
+
+    '''
+    This method will add additional security groups to instance
+    '''
+    def add_security_group_to_instance(self, instance_id, sgid):
+        try:
+            self.servers_client.add_security_group(instance_id, name=sgid)
+            LOG.debug("Added security group {} to instance {}".format(sgid, instance_id))
+            return True
+        except AssertionError as e:
```

Comment:
In case of any exception, does the servers_client call raise only AssertionError ? What if any other type of exception is raised ?

Reply:
Changed to capture generic exception.